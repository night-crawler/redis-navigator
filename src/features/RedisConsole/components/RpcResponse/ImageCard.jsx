import PropTypes from 'prop-types';
import React from 'react';
import { Image } from 'semantic-ui-react';


ImageCard.propTypes = {
  dataUri: PropTypes.string.isRequired,
};
export default function ImageCard(props) {
  const { dataUri } = props;
  return <Image className='ImageCard' src={ dataUri } size='medium' />;
}
