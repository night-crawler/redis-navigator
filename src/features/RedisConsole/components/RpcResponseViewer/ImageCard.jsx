import PropTypes from 'prop-types';
import React from 'react';
import { Image } from 'semantic-ui-react';


ImageCard.propTypes = {
  dataUri: PropTypes.string.isRequired,
};
export function ImageCard(props) {
  return <Image 
    className='ImageCard' 
    src={ props.dataUri } 
    size='medium' 
  />;
}
