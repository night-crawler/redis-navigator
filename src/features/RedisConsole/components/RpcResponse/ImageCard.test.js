/* eslint-disable react/react-in-jsx-scope,no-undef */
import ImageCard from './ImageCard';


describe('ImageCard', () => {
    it('should render ImageCard', () => {
        const wrapper = shallow(<ImageCard dataUri='1' />);
        expect(wrapper).toMatchSnapshot();
    });
});
