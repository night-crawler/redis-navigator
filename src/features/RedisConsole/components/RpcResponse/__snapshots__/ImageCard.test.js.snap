// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ImageCard should render ImageCard 1`] = `
<Image
  as="img"
  size="medium"
  src="1"
  ui={true}
/>
`;
